versions/3.0.md

```diff
@@ -85,10 +85,10 @@ Version | Date | Notes
 Path templating refers to the usage of curly braces ({}) to mark a section of a URL path as replaceable using path parameters.
 
 ##### <a name="mediaTypes"></a>Media Types
-Media type definitions are spread across several resources.
-The media type definitions should be in compliance with [RFC 6838](http://tools.ietf.org/html/rfc6838).
+Media-type definitions are spread across several resources.
+The media-type definitions should be in compliance with [RFC 6838](http://tools.ietf.org/html/rfc6838).
```

Comment:
leave as "Media type" with a space, not a hyphen - this is how it is used on http://tools.ietf.org/html/rfc6838 !!